src/main/src/main/java/org/geoserver/security/impl/LayerGroupContainmentCache.java

```diff
@@ -72,7 +77,7 @@ public LayerGroupContainmentCache(Catalog catalog) {
         buildLayerGroupCaches();
     }
 
-    private void buildLayerGroupCaches() {
+    public void buildLayerGroupCaches() {
```

Comment:
Looks to me that this method can be left as private as is no more needed to have it public.